Core/include/Acts/Vertexing/AdaptiveMultiVertexFitter.ipp

```diff
@@ -217,30 +227,17 @@ Acts::AdaptiveMultiVertexFitter<input_track_t, linearizer_t>::
     setAllVertexCompatibilities(
         State& state, Vertex<input_track_t>* currentVtx,
         const VertexingOptions<input_track_t>& vertexingOptions) const {
-  VertexInfo<input_track_t>& currentVtxInfo = state.vtxInfoMap[currentVtx];
+  // Update the 3D impact parameters of all tracks
+  updateImpactParams3D(state, currentVtx, vertexingOptions);
```

Comment:
Is this why the call is not needed up top anymore?

Reply:
Exactly!